src/utils/interpolate.hpp

```diff
@@ -0,0 +1,134 @@
+#ifndef INTERPOLATE_HPP_
+#define INTERPOLATE_HPP_
+//========================================================================================
+// Parthenon performance portable AMR framework
+// Copyright(C) 2022 The Parthenon collaboration
+// Licensed under the 3-clause BSD License, see LICENSE file for details
+//========================================================================================
+//! \file interpolate.hpp
+//  \brief Interpolation using cubic Hermite polynomials
+//
+// Performs monotone interpolation using cubic Hermite polynomials.
+// Reference: "Monotone piecewise cubic interpolation," SIAM J. Numer. Anal.,
+//            Vol. 17, No. 2, April 1980. [https://doi.org/10.1137/0717021]
+//========================================================================================
+
+#include "../kokkos_abstraction.hpp"
+#include <ostream>
+
+template <class VectorContainer>
```

Comment:
I would put this class in a `namespace parthenon {}` for consistency with the rest of `utils`

Reply:
fixed